wdl-lint/src/rules/inconsistent_newlines.rs

```diff
@@ -22,7 +22,7 @@ fn inconsistent_newlines(span: Span) -> Diagnostic {
     Diagnostic::note("inconsistent newlines detected")
```

Comment:
Perhaps change these to line endings too, as that is a bit more focused (for example, that rules out newline literal characters). That being said, if you want to keep newlines for both this comment and the one below, I'm fine with that.